### PR TITLE
Index - Advertise OT applications.

### PR DESCRIPTION
**Pull request by jomey**
To go along with the email announcement

### CODE REVIEW

**Review by JessicaS11 — APPROVED** [review]
I made a few other edits to get the environment to solve and note that it was the 5th (not 3rd) for IS2. ; ) Builds fine locally and should be good to go!

File: conda/environment.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ dependencies:
   - cookiecutter
   - git
   - ipyleaflet
-  - jupyter-book=0.13
+  - jupyter-book>=0.13
```
Comment (jomey) [inline conda/environment.yml:13]: Just a note that the splash page is not using any components of JupyterBook. This long list of libraries is a relic of cloning the big website template and could use some scrubbing to make it tailored to the standalone splash page.

**Comment by jomey** [comment]
> I made a few other edits to get the environment to solve and note that it was the 5th (not 3rd) for IS2. ; ) Builds fine locally and should be good to go!

Thanks for catching that. The "blind copy cat" struck again :smile:

**Comment by aaarendt** [comment]
my commit changes the index.html file to match the one @jomey has for SnowEx. Needed to do this to deploy locally as it was throwing an error (it was looking for the 'apply' attribute in the yaml file which is temporarily commented out). We should make sure this is reflected in the template at some point.

File: cookiecutter.yaml [hunk 2]
```diff
@@ -18,21 +28,21 @@ about:
...
           title: 2019 Hackweek
           new_window: true
-apply:
-  url: '#'
-  title: Application coming soon
-hackweek_mission: https://escience.washington.edu/using-data-science/hackweeks
+# apply:
+#   url: '#'
+#   title: Application coming soon
+# hackweek_mission: https://escience.washington.edu/using-data-science/hackweeks
```
Comment (jomey) [inline cookiecutter.yaml:23]: Should we keep the button in there for advertisement purposes?

**Comment by jomey** [comment]
> my commit changes the index.html file to match the one @jomey has for SnowEx. Needed to do this to deploy locally as it was throwing an error (it was looking for the 'apply' attribute in the yaml file which is temporarily commented out). We should make sure this is reflected in the template at some point.

Part of a pending PR https://github.com/uwhackweek/jupyterbook-template/pull/143

File: conda/environment.yml (hunk 1 above)
Comment (JessicaS11) [reply conda/environment.yml:13]: > This long list of libraries is a relic of cloning the big website template and could use some scrubbing to make it tailored to the standalone splash page.

Agreed!

File: cookiecutter.yaml (hunk 2 above)
Comment (JessicaS11) [reply cookiecutter.yaml:23]: I didn't want to leave it there and end up with folks trying to apply to the OT getting to the website and seeing the ambiguous "application coming soon" text... So I'd say keep it and add "Participant" or leave it out. Perhaps we can instead put it as an "application coming soon" button in the info for participants section?

**Review by aaarendt — APPROVED** [review]

File: cookiecutter.yaml (hunk 2 above)
Comment (aaarendt) [reply cookiecutter.yaml:23]: Agree with Jessica and noting that my latest changes include an option for participants to signal their interest via our form.

**Pull request merged**